### PR TITLE
Add icon for NULL SymbolKind

### PR DESCRIPTION
**Pull request by gerdleon**
Modified the "class.svg" icon.

### CODE REVIEW

**Comment by mickaelistria** [comment]
I'm not so sure about the icon to represent null. What about something like Ø ?

**Comment by gerdleon** [comment]
> I'm not so sure about the icon to represent null. What about something like Ø ?

Yes, let's do that.

File: org.eclipse.lsp4e/src/org/eclipse/lsp4e/ui/LSPImages.java [hunk 1]
```diff
@@ -34,219 +34,222 @@
 
 public final class LSPImages {
 
-	private LSPImages() {
```
Comment (rubenporras) [inline org.eclipse.lsp4e/src/org/eclipse/lsp4e/ui/LSPImages.java:37]: @gerdleon , it looks like the line tabs are now spaces, could you convert them back to tabs?

Comment (gerdleon) [reply org.eclipse.lsp4e/src/org/eclipse/lsp4e/ui/LSPImages.java:37]: > I'm also not sure deriving null icon from the class one makes a lot of sense: null is not about defining something, but rather an undefined reference, or a reference to nothing, IMO, the simplest would be to put the symbol on a transparent background, similarly to number or string or boolean icons which are about "native" concepts.

Makes sense.

**Comment by gerdleon** [comment]
I will look into a better icon.

**Comment by mickaelistria** [comment]
thank you!

**Pull request merged**